### PR TITLE
Highlight ideas

### PR DESCRIPTION
**Pull request by niklasschloegel**
According to issue https://github.com/bykof/what-to-code/issues/2, these changes add the ability to highlight ideas when liking.

### CODE REVIEW

File: frontend/hooks/useLikesCookie.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { useCookies } from "react-cookie";
+
+export default () => {
+    const [cookies, setCookie] = useCookies(["likes"]);
+
+    var inTenYears = new Date();
```
Comment (bykof) [inline frontend/hooks/useLikesCookie.js]: please do not use `var` anymore. Just use `const`

File: frontend/hooks/useLikesCookie.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    inTenYears.setFullYear(inTenYears.getFullYear() + 10);
+
+    let savedLikes = cookies.likes ? cookies.likes : [];
```
Comment (bykof) [inline frontend/hooks/useLikesCookie.js]: You do not rewrite this variable, so you could just set it to const :)

File: frontend/hooks/useLikesCookie.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    if (!cookies.likes) {
+        updateCookie();
+    }
+
+    const isLiked = (id) => {
+        return savedLikes.includes(id);
+    };
+
+    const updateLikes = (id) => {
```
Comment (bykof) [inline frontend/hooks/useLikesCookie.js]: The naming `updatesLikes` inherits, that you will update more than one like. 
Maybe use `like(id)`

File: frontend/hooks/useLikesCookie.js (hunk 2 above)
Comment (bykof) [inline frontend/hooks/useLikesCookie.js]: Better store the likes in an object... 
you could store the id of the idea in the object as key and just the value `true` as object-value. 
Then your access becomes more perfomant.

`const likesPerIdea = {};` 

and in the function: 
```
const toggleLike = (id) {
  if (id in likesPerIdea) {
    likesPerIdea[id]  = true;
  } else {
    delete likesPerIdea[id];
  }
}
```

in isLike you can just use now: 

```
const isLiked = (id) =>
  return (id in likesPerIdea)
};
```

Comment (bykof) [reply frontend/hooks/useLikesCookie.js]: Here is the benchmark of objects vs arrays: https://jsben.ch/Y9jDP

File: frontend/hooks/useLikesCookie.js (hunk 1 above)
Comment (niklasschloegel) [reply frontend/hooks/useLikesCookie.js]: I'll change it in useTokenCookie.js too 👍🏼

Comment (bykof) [reply frontend/hooks/useLikesCookie.js]: Perfect thanks! was probably a copy pasta failure

**Review by bykof — APPROVED** [review]

**Pull request merged**